createGraph.go

```diff
@@ -49,7 +49,17 @@ func GraphHandler(s *discordgo.Session, i *discordgo.InteractionCreate) {
 	log.Println("creating graph...")
 	pngs := make([]io.Reader, 0, len(dataSet[0].Items))
 	for title := range dataSet[0].Items {
-		png, err := createPngGraph(dataSet, title)
+		y_min, y_max = "", ""
```

Comment:
y_min, y_maxはまだ定義されてないから=で代入はできないよ。
```suggestion
		y_min, y_max := "", ""
```
か
```suggestion
		var y_min, y_max string
```
にする必要がある

Reply:
あとここはintにしてもいいかも。まあ今のところどっちでもいいから必要に応じて変更すればいいか。